index.js

```diff
@@ -150,3 +150,48 @@ cardButton.forEach((each) => each.addEventListener('click', function OpenPopUp()
     overlay.classList.add('disable');
   });
 }));
+
+// Form Validation
+const form = document.querySelector('.form-input');
+const email = document.getElementById('email');
+
+// Error Mesaage for Invalid email
+const setError = (element, message) => {
+  const inputField = element.parentElement;
+  const errorDisplay = inputField.querySelector('.error');
+  errorDisplay.innerText = message;
+  inputField.classList.add('error');
+  inputField.classList.remove('success');
+};
+
+// Success Mesaage for Valid email
+const setSuccess = (element) => {
+  const inputField = element.parentElement;
+  const errorDisplay = inputField.querySelector('.error');
+  errorDisplay.innerText = '';
+  inputField.classList.add('success');
+  inputField.classList.remove('error');
+};
+
+// Email order check
+const isValidEmail = (email) => {
+  const re = /^([a-z-]+)@([a-z\d]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+  return re.test(email);
+};
+
+const validateInputs = () => {
+  const emailValue = email.value.trim();
+
+  if (emailValue === '') {
+    setError(email, 'Email is required');
+  } else if (!isValidEmail(emailValue)) {
+    setError(email, 'Provide a valid email address');
+  } else {
+    setSuccess(email);
+  }
+};
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  validateInputs();
+});
```

Comment:
- [ ] Even though I only used lowercase characters when entering an email, your form is still showing that it is incorrect. Please ensure If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.